### ISSUE TITLE
Repository is missing assets and resources

### ISSUE THREAD

**Issue by bfabio** [italia/cie-ideaapp#15]
The repository is missing assets, resources files, the manifest, etc., which are needed to develop and compile the application.

I didn't fire up Android Studio yet, but I don't see anything in `MainActivity.java` that can load the UI layout, even if the resource files were there. That makes me think the code is from an earlier version.

**Comment by matpag** [comment]
Even if we copy the classes in a new Android project, it will not compile because there are missing referenced classes.

By the way I've found this undocumented repository here: https://github.com/italia/cieid-android-sdk
which seems at least to compile (even if it is not the same thing)

**Comment by bfabio** [comment]
@matpag Yes, that repo is the way to go since it's active and it supersedes this one in features.

It's still a bummer though we don't have the code for IDEA App. :disappointed:
